### PR TITLE
LPS-176748 [PRM] Changing activity status according mdf request status

### PR DESCRIPTION
**Pull request by biellopes00**

### CODE REVIEW

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/extra/remote-app/src/common/utils/constants/status.ts [hunk 1]
```diff
@@ -57,4 +57,8 @@ export const Status = {
 		key: 'moreInfoRequested',
 		name: 'More Info Requested',
 	},
+	SUBMITED: {
+		key: 'submited',
```
Comment (thaynam) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/extra/remote-app/src/common/utils/constants/status.ts:61]:
```suggestion
		key: 'submitted',
```

**Comment by thaynam** [comment]
the typos are breaking the deploy, always test before sending the PR

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/fragments/group/src/mdf-request-detail-collection/fragments/activities-list-panel/index.js [hunk 2]
```diff
@@ -152,15 +152,15 @@ const TypeActivityKey = {
 };
 
 const ActivityStatus = {
-	ACTIVE: 'active',
 	APPROVED: 'approved',
 	CLAIMED: 'claimed',
 	EXPIRED: 'expired',
+	SUBMITED: 'submited',
```
Comment (jairmedeiros) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/fragments/group/src/mdf-request-detail-collection/fragments/activities-list-panel/index.js:158]:
```suggestion
	SUBMITTED: 'submitted',
```

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/fragments/group/src/mdf-request-detail-collection/fragments/activities-list-panel/index.js [hunk 3]
```diff
@@ -152,15 +152,15 @@ const TypeActivityKey = {
...
 	UNCLAIMED: 'unclaimed',
 };
 
 const activityStatusClassName = {
-	[ActivityStatus.ACTIVE]: 'label label-tonal-success ml-2',
+	[ActivityStatus.SUBMITED]: 'label label-tonal-warning ml-2',
```
Comment (jairmedeiros) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/fragments/group/src/mdf-request-detail-collection/fragments/activities-list-panel/index.js:163]:
```suggestion
	[ActivityStatus.SUBMITTED]: 'label label-tonal-warning ml-2',
```

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/object-definitions/activity.json [hunk 4]
```diff
@@ -377,6 +377,14 @@
 					"name": "stateFlow",
 					"value": {
 						"objectStates": [
+							{
+								"key": "submited",
```
Comment (jairmedeiros) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/object-definitions/activity.json:381]:
```suggestion
								"key": "submitted",
```

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/object-definitions/activity.json [hunk 5]
```diff
@@ -364,7 +364,7 @@
 		{
 			"DBType": "String",
 			"businessType": "Picklist",
-			"defaultValue": "approved",
+			"defaultValue": "submited",
```
Comment (jairmedeiros) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/object-definitions/activity.json:367]:
```suggestion
			"defaultValue": "submitted",
```

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/list-type-definitions/activity-status.list-type-entries.json [hunk 6]
```diff
@@ -1,4 +1,12 @@
 [
+	{
+		"key": "submited",
+		"name": "Submited",
+		"name_i18n": {
+			"en-US": "Submited"
```
Comment (jairmedeiros) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/list-type-definitions/activity-status.list-type-entries.json:6]:
```suggestion
		"key": "submitted",
		"name": "Submitted",
		"name_i18n": {
			"en-US": "Submitted"
```

**Review by jairmedeiros — CHANGES_REQUESTED** [review]

**Review by thaynam — CHANGES_REQUESTED** [review]

**Pull request closed without merging**